site/src/site/sphinx/en/tutorial-for-beginners.md

```diff
@@ -0,0 +1,282 @@
+## A Tutorial for Beginners
+
+### Prepare the demo
+
+Save the following demo to `demo/Demo.java` to have the first try on Arthas. 
+
+<sub>(If you are using Unix/Linux/Mac, then the commands below can be used.)</sub>
+
+```bash
+# you'd better find a proper location for the tutorial
+$ mkdir demo && touch demo/Demo.java
+
+# if there are indentation issues
+# ":set paste" in vim command mode to enter insert & paste mode
+$ vim demo/Demo.java 
+# ":set nopaste" afterwards to reset
+# ":wq" to save and exit
+```
+
+Copy and paste the following code to `demo/Demo.java`:
+
+```java
+package demo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+public class Demo {
+    static class Counter {
+        private static AtomicInteger count = new AtomicInteger(0);
+        public synchronized static void increment() {
```

Comment:
count is already atomic and thread safe，not sure why we still need synchronized keyword?